### ISSUE TITLE
Pull Request Dashboard

### ISSUE THREAD

**Issue by github-product-roadmap** [github/roadmap#828]
### Summary

Customizable version of https://github.com/pulls designed to help users quickly see which pull requests need their attention. Users today must click through multiple tabs and continually refresh the page to get the same list, often resulting in delays and inconvenience on other users that must continually "ping" other users to review or perform other actions on their pull requests.

### Intended Outcome

1. Help users quickly tell which pull requests are waiting on them
2. Reduce the avoidable delays that inevitably happen in every pull request that slow or block progress
3. Higher productivity for individual contributors and teams

### How will it work?

When a user navigates to https://github.com/pulls, they will see a consolidated view of the pull requests needing attention. This view can be easily filtered or adjusted and saved as a custom view that can be returned to later. This view is also updated on the fly to ensure the latest status is presented so the user can stay on top of the pull requests they are involved in.

**Comment by ankneis** [comment]
This issue is being closed as outdated. For more information, please check out this [Discussion post](https://github.com/github/roadmap/discussions/1014). Stay tuned for new additions to our refreshed public roadmap!

**Comment by RokeJulianLockhart** [comment]
> This issue is being closed as outdated. For more information, please check out this [Discussion post](https://github.com/github/roadmap/discussions/1014). Stay tuned for new additions to our refreshed public roadmap!

@ankneis, you really should provide specific rationale for why you've closed each issue, since some of them are incredibly popular, and continue to appear to make significant sense (to me, at least).

**Comment by ankneis** [comment]
We wanted to provide more details on why this issue was removed from the public roadmap. An improved experience for finding pull requests that most need attention is currently in an exploratory phase, so we have removed it from this roadmap while we develop more definition and certainty on our scope and timelines. 

Here are a number of feedback discussions that are similar to this topic—feel free to upvote and share your feedback where it fits best, or create a new discussion if needed! Your input really helps us shape our next steps.

- https://github.com/orgs/community/discussions/42848
- https://github.com/orgs/community/discussions/12449
- https://github.com/orgs/community/discussions/9974
- https://github.com/orgs/community/discussions/12447
